--- lib/cloudregister/registerutils.py
+++ lib/cloudregister/registerutils.py
@@ -339,6 +339,8 @@ def get_register_cmd():
     """Determine which command we need to use to register the system"""
 
     register_cmd = '/usr/sbin/SUSEConnect'
+    # Return early on SLE12
+    return register_cmd
     if is_transactional_system():
         cmd_name = 'transactional-update'
         for path in ['/sbin/', '/usr/sbin/']:
