diff --git a/cloudregister/smt.py b/cloudregister/smt.py
index 62c461a..19c7ebd 100644
--- a/cloudregister/smt.py
+++ b/cloudregister/smt.py
@@ -128,6 +128,7 @@ class SMT:
     # --------------------------------------------------------------------
     def get_ipv6(self):
         """Return the IP address"""
+        return None
         # Before handling ipv6 the IP address was stored in the _ip
         # member. When the SMT object is restored from an old pickeled
         # file the _ipv6 member does not exist. Handle this transition
